src/pip/_internal/commands/debug.py

```diff
@@ -110,6 +133,11 @@ def run(self, options, args):
         show_value('sys.platform', sys.platform)
         show_sys_implementation()
 
+        show_value("REQUESTS_CA_BUNDLE", os.environ.get('REQUESTS_CA_BUNDLE'))
+        show_value("CURL_CA_BUNDLE", os.environ.get('CURL_CA_BUNDLE'))
+        show_value("pip._vendor.certifi.where()", where())
+        show_value("'cert' config value", ca_bundle_info(self.parser.config))
```

Comment:
@chrahunt Does this look OK to you?

This doesn't include the filename we're loading from, since, well that's tricky to get from the `Configuration` object currently (we're not keeping track of which property came from where).

Reply:
Yeah it would be fine, honestly there's not that many it *could* be coming from, so not really needed in hindsight.